onnxruntime/core/providers/nuphar/scripts/symbolic_shape_infer.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, int_max, auto_merge, guess_output_rank, verbose):
             'Div'                   : self._infer_symbolic_compute_ops,
             'Expand'                : self._infer_Expand,
             'Equal'                 : self._infer_symbolic_compute_ops,
+            'Floor'                 : self._pass_on_sympy_data,
```

Comment:
>'Floor'                 : self._pass_on_sympy_data [](start = 12, length = 50)

If the input is floating point, we should use sympy.Floor, as the op might change sympy data instead of just pass on

Reply:
Fixed. Thanks.